.github/workflows/cdci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         run: |
           cd docs
           vuegen -dir example_data/Earth_microbiome_vuegen_demo_notebook -rt pdf -qt_checks
-          vuegen -c example_data/Earth_microbiome_vuegen_demo_notebook/Earth_microbiome_vuegen_demo_notebook_config.yaml -rt pdf
+          vuegen -c example_data/Earth_microbiome_vuegen_demo_notebook/Earth_microbiome_vuegen_demo_notebook_config.yaml -rt pdf -qt_checks
```

Comment:
so here it is needed to install tinytex on the fly, right? but just not in the docker image? 

It's good that this has to be activated, so that updates do not just happen. but we should document this as well.